docs/preview/counterExpression.md

```diff
@@ -0,0 +1,39 @@
+# Counter Expression
+
+## Overview
```

Comment:
Is a counter scoped to a definition? Or can two definitions the same counter?

Reply:
It's per-definition. We don't share variables across definitions. Assuming we implement top-level variables in YAML files, it will be the same thing. The variables may live in the `.yml`, but the build definition is what owns the final value.

Reply:
I've addressed this in the Overview section.